spec/controllers/v0/users_controller_spec.rb

```diff
@@ -149,4 +149,26 @@
       end
     end
   end
+
+  describe '#credential_emails' do
+    let(:user) { create(:user, :loa1) }
+    let(:user_account) { create(:user_account, user: user) }
+
+    before do
+      sign_in user
+      create(:user_verification, idme_uuid: user.idme_uuid, user_credential_email: create(
```

Comment:
I would probably move this into a `let` variable, and the `user_credential_email` into a let variable. It will let you refer to the value in your `expected_response` so you don't have to copy the value in multiple places. Looks like this:
`let(:idme_user_verification) { create(:user_verification, idme_uuid: user.idme_uuid, user_credential_email: }`
`let(:user_credential_email) { create(:user_credential_email, credential_email: 'email1@example.com') }`
`let(:expected_response) { {idme_user_verification.credential_type => user_credential_email.credential_email } }`